helicone-helpers/manual_logger/HeliconeManualLogger.ts

```diff
@@ -0,0 +1,178 @@
+export class HeliconeManualLogger {
+  private apiKey: string;
+  private headers: Record<string, string>;
+  private readonly LOGGING_ENDPOINT: string =
```

Comment:
would be great to have this configurable but defaulted on the constructor

Reply:
Hey @chitalian , I didn't understand this one - isn't this being done
https://github.com/Helicone/helicone/blob/ce8085630010a05acc64907b8b9d70798994a935/helicone-helpers/manual_logger/HeliconeManualLogger.ts#L9-L12

Reply:
specifically the logging endpoint